packages/playground-examples/copy/ko/TypeScript/Primitives/Any.ts

```diff
@@ -0,0 +1,50 @@
+// Any는 TypeScript의 탈출 구문입니다.
+// 코드의 일부를 동적이고 JavaScript처럼 선언하고 싶을 때,
+// 또는 타입 체계의 제한을 벗어나고 싶을 때
+// any를 사용할 수 있습니다.
+
+// JSON 파싱은 any의 좋은 예시입니다.
+
+const myObject = JSON.parse("{}");
+
+// Any는 여러분이 코드를 더 잘 알고 있음을 뜻하기 때문에,
+// 엄격하게 true가 아니더도 TypeScript는 여러분의 코드가 안전하다고 믿게 됩니다.
+// 예를 들어, 아래 코드는 오류가 발생할 것입니다.
+
+myObject.x.y.z;
+
+// any를 사용함으로써 타입의 안전성을 희생하는 대신
+// 코드를 원래의 JavaScript에 더욱 가깝게 작성할 수 있습니다.
+
+// any는 여러분이 어떤 타입(never를 제외하고)이든
+// 하나의 타입을 다른 타입에 할당할 수 있게끔 대신할 수 있는
```

Comment:
```suggestion
// 다른 타입에 할당할 수 있도록 만드는
```
좀 더 자연스럽게 중복되는 의미를 제거해봤습니다 :)
[제안]